site/content/en/_index.html

```diff
@@ -19,19 +19,19 @@
 </div>
 {{< /blocks/cover >}}
 
-{{< blocks/section >}}
+{{< blocks/section type="row" >}}
 
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/kubernetes-sigs/cloud-provider-azure" %}}
+{{% blocks/feature icon="fa-brands fa-github" title="Contributions welcome!" url="/" %}}
```

Comment:
Is it true that we don't need the repo url now?

Reply:
updated. Thanks!